### DISCUSSION TITLE
How do I debug spawned machines?

### DISCUSSION THREAD

**Discussion by pckilgore** [statelyai/xstate#2414] [Q&A] [answered]
I don't see a way of passing `{ debug: true }` or equivalent into a machine spawned by another machine.  I'd like to see that child machine in @xstate/inspect.

**Comment by johtso** [comment, marked as answer]
They should inherit the debug setting, and you should be able to see them in the machine drop down list on the right hand side.

**Reply by pckilgore** [reply]
Ahh! This (inherited) page variously spawns nine machines, and I was looking for it under the `{ name: string }` option passed to spawn, but it shows up under the `id` defined in the machine.

Thanks @johtso !
